File: omninavi/src/main/java/com/omni/omninavi/model/google_navigation/GoogleBounds.java
package com.omni.omninavi.model.google_navigation;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

/**
 * Created by wiliiamwang on 14/06/2017.
 */

public class GoogleBounds implements Serializable {

    @SerializedName("northeast")
    private OmniLatLng northeast;
    @SerializedName("southwest")
    private OmniLatLng southwest;

    public OmniLatLng getNortheast() {
        return northeast;
    }

    public void setNortheast(OmniLatLng northeast) {
        this.northeast = northeast;
    }

    public OmniLatLng getSouthwest() {
        return southwest;
    }

    public void setSouthwest(OmniLatLng southwest) {
        this.southwest = southwest;
    }
}

File: navisdk/src/main/java/com/omni/navisdk/module/google_navigation/GoogleBounds.kt
package com.omni.navisdk.module.google_navigation

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class GoogleBounds : Serializable {
    @SerializedName("northeast")
    var northeast: OmniLatLng? = null

    @SerializedName("southwest")
    var southwest: OmniLatLng? = null

}